### DISCUSSION TITLE
Netbox on RHEL9 with httpd; 503 and I can't find an error

### DISCUSSION THREAD

**Discussion by bowlweevil** [netbox-community/netbox#16063] [Help Wanted!]
Pretty much what the title says; trying to install Netbox on RHEL9 (i do not have a choice on OS).
Found a few different guides and issue resolutions scattered around the internets but still can't get it to work. 
Netbox Dev environment works when launched from venv.
Netbox, netbox-rq and httpd all launch without errors as services. 
503 error page presents the correct SSL cert defined in netbox config, so apache is handing off to netbox fine.
No errors in sql logs.

Netbox installed via official docs, except:

firewalld ports added TCP 8000,8001,443,80
SElinux disabled
chown /opt/netbox to netbox:netbox
using godaddy ssl cert (renamed netbox.crt/key for ease)

Logging enabled via this guide: https://majornetwork.net/2022/09/configuring-logging-in-netbox/
Log level set to debug.
99pct sure I'm missing something obvious but not sure where to look. Logs don't show much, no obvious errors.

Any ideas where i could look or what I'm missing?

Thank you for any/all help!

**Comment by markkuleinio** [comment]
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes: 

> 503 Service Unavailable
    The server cannot handle the request (because it is overloaded or down for maintenance).

The error message is coming from Apache, I'd say. Thus, NetBox logging is of no help because the request is never handled by NetBox as Apache was not able to forward the request to it.

```
sudo systemctl status netbox
sudo journalctl -eu netbox
ss -ntl
```

Also check that your Apache is configured to use the same port (8000? 8001?) that your Gunicorn/uWSGI is listening to.
